make-package-lock.py

```diff
@@ -45,11 +45,11 @@ def create_download_info(
     return result
 
 
-def index_download_checksums() -> T.Dict[str, DownloadInfo]:
+def index_download_checksums(dir_path: str) -> T.Dict[str, DownloadInfo]:
```

Comment:
Change the argument type to `pathlib.Path` and use `dir_path.rglob("*.hash")` below